manifests/install.pp

```diff
@@ -3,9 +3,19 @@
   assert_private()
 
   $download_url = $::chocolatey::chocolatey_download_url
-  $unzip_type   = $::chocolatey::use_7zip ? {
-    true      => '7zip',
-    default   => 'windows'
+  $sevenzip_exe = "${::system32}\\chocolatey_temp_7za.exe"
```

Comment:
If you don't mind code style, we could use something like
```ruby
$sevenzip_exe = inline_template('"#{ENV['TEMP']}\\7za.exe"')
```
Witch is basically the current behavior.

Reply:
That could work, but I wonder what the Puppet style guide says about that.